src/main/java/our/fashionablesimba/yanawa/user/exception/EmailDuplicationException.java

```diff
@@ -0,0 +1,10 @@
+package our.fashionablesimba.yanawa.user.exception;
+
+import our.fashionablesimba.yanawa.user.domain.Email;
+
+import javax.validation.Valid;
+
+public class EmailDuplicationException extends RuntimeException {
+    public EmailDuplicationException(@Valid Email email) {
```

Comment:
컨트롤러에서만 유효성을 체크하는게 좋을 것 같습니다 ☺️ 

```suggestion
    public EmailDuplicationException(Email email) {
```